packages/react/src/accordion-trigger/AccordionTrigger.tsx

```diff
@@ -4,43 +4,51 @@ import { type ReactNode, forwardRef } from "react";
 import { Box, type BoxProps } from "../box";
 import { Flex } from "../flex";
 import { IconAngleDown } from "../icons/IconAngleDown";
+import { IconAngleRight } from "../icons/IconAngleRight";
 import { extractSprinkles } from "../sprinkles";
 import * as styles from "./AccordionTrigger.css";
 
 type AccordionTriggerProps = BoxProps<
   typeof RadixAccordion.Trigger,
   {
     endDecorator?: ReactNode;
+    position?: "end" | "start";
```

Comment:
having both start/end decorations and a `position` prop does not make sense here. we also should not allow toggling the position at the individual trigger level - rather this should be an appearance or variant prop at the very top `<Accordion />` component level

for now let's change this to an `appearance` prop and call it `primary` (chevron at the start) and `secondary` (chevron at the end)

we can use context to pass this appearance prop down to the triggers and then inside the trigger if a `startDecorator` or `endDecorator` is explicitly given we'll use that, else we'll fallback to `IconAngleDown` at the appropriate position.